graphql-dxm-provider/src/main/resources/META-INF/configurations/org.jahia.modules.graphql.provider-default.cfg

```diff
@@ -33,3 +33,4 @@
 # List of allowed origin for CORS access :
 #
 # http.cors.allow-origin=http://mysite1.com, http://mysite2.com
+graphql.fields.node.limit = 5000
```

Comment:
Maybe add a comment as to what this setting does ?

Reply:
fixed